go/cmd/vtctldclient/command/reparents.go

```diff
@@ -292,6 +294,7 @@ func init() {
 	Root.AddCommand(InitShardPrimary)
 
 	PlannedReparentShard.Flags().DurationVar(&plannedReparentShardOptions.WaitReplicasTimeout, "wait-replicas-timeout", topo.RemoteOperationTimeout, "Time to wait for replicas to catch up on replication both before and after reparenting.")
+	PlannedReparentShard.Flags().DurationVar(&plannedReparentShardOptions.TolerableReplicationLag, "tolerable-replication-lag", 0, "Amount of replication lag that is considered acceptable for a tablet to be considered when Vitess makes the choice of a new primary.")
```

Comment:
Yeah, I thought that having the behavior opt-in is pretty good so that we are not forcing anyone to use it. But yes, I am with on the 0 representing infinite conundrum. The way I reasoned about it was that there really is no good way of specifying infinite acceptable replication lag, and 0 acceptable replication lag doesn't make any sense either, so I just used 0 to represent the unspecified case (but yes, it is functionally the same as specifying an infinite value.)